### PR TITLE
add balena, aws and gh CLIs

### PR DESCRIPTION
**Pull request by ab77**
change-type: patch

### CODE REVIEW

File: runner/jammy/amd64/eget.toml [hunk 1]
```diff
@@ -48,3 +48,11 @@ file = "*"
...
 target = "/"
 file = "*"
+
+["balena-io/balena-cli"]
+asset_filters = ["linux", "x64", "standalone", ".zip"]
+tag = "v15.2.0"
+target = "/usr/local/bin/balena"
+verify_sha256 = "789237a919fda6c735c78b7f1003a6277909093d00809b643282653cdc0c85ec"
+file = "balena-cli/balena"
+all = false
```
Comment (klutchell) [inline runner/jammy/amd64/eget.toml:58]: Does this actually work? IIRC the entire contents of the tarball are required, and a symlink to the binary.

Maybe we need a test in the Dockerfile or the docker-compose tests to check that `balena version` can be executed for x86 builds.

Comment (ab77) [reply runner/jammy/amd64/eget.toml:58]: Good idea, I've added the tests. On a related note, we really need to get our sh*t together and publish linux/arm balena CLI release.

Comment (ab77) [reply runner/jammy/amd64/eget.toml:58]: > Does this actually work? IIRC the entire contents of the tarball are required, and a symlink to the binary.

You are right, i thought `balena` was a binary, but it's just a script. I need to figure out where to put it, without adding to PATH.

Comment (klutchell) [reply runner/jammy/amd64/eget.toml]: IIRC you can extract ALL files to `/usr/local/bin/balena-cli` and in the Dockerfile add a symlink `/usr/local/bin/balena -> ./balena-cli/balena` or something like that.

Comment (ab77) [reply runner/jammy/amd64/eget.toml]: extracting the contents on `balena-cli/*` into `/usr/local/bin/` seems to work...

Comment (klutchell) [reply runner/jammy/amd64/eget.toml]: I expect that is putting a lot of extra stuff into the PATH that we don't want. Try it locally and see.

File: runner/Dockerfile [hunk 2]
```diff
@@ -74,6 +79,10 @@ RUN --mount=type=secret,id=GITHUB_TOKEN \
 
 RUN chown -R github:github /home/github
 
+# no GitHub linux/arm release(s)
+RUN [ "$(uname -m)" == x86_64 ] && \
+    (ln -s /usr/local/lib/balena-cli/balena /usr/local/bin/balena && balena version) || true
```
Comment (klutchell) [inline runner/Dockerfile:84]: Are you certain that `condition && command || command` actually acts as an if|else? I recall reading on shellcheck documentation that it acts more like `condition && (command || command)` but I could be wrong.

You could just put it in a proper if block to be sure and avoid `|| true` altogether.

**Comment by klutchell** [comment]
Lgtm

**Comment by ab77** [comment]
lgtm

**Pull request merged**